projects/components/src/multi-select/multi-select.component.scss

```diff
@@ -92,7 +92,7 @@
   flex-direction: column;
 
   .divider {
-    padding: 0px 16px 4px 16px;
+    padding: 0 16px 4px 16px;
```

Comment:
why remove px ?

Reply:
zeroes don't need px :)

Reply:
so this is just a nit change?

Reply:
yea. this is just a minor cleanup since I was in the area :)